sandbox/lua/modules/line_protocol.lua

```diff
@@ -0,0 +1,246 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The module supports Graphite stat format in addition to InfluxDB... maybe 'stat_formatting' or something similar?